java/test/src/main/java/io/ray/test/RuntimeEnvTest.java

```diff
@@ -161,4 +171,23 @@ public void testPerTaskEnvVarsOverwritePerJobEnvVars() {
       Ray.shutdown();
     }
   }
+
+  public void testJarsInActor() {
+    try {
+      Ray.init();
```

Comment:
Because other cases in this test file test the job level configs, we should set different config before Ray.init(). You can view the whole file for details.